Cython/Utility/Builtins.c

```diff
@@ -257,7 +257,7 @@ static PyObject *__Pyx_PyLong_AbsNeg(PyObject *n) {
     {
         PyObject *copy = _PyLong_Copy((PyLongObject*)n);
         if (likely(copy)) {
-            Py_SIZE(copy) = -(Py_SIZE(copy));
+            __Pyx_SET_SIZE(copy, -Py_SIZE(copy));
```

Comment:
Since this seems a lot less clear now, I think it's worth a comment.
```suggestion
            // negate the size to swap the sign
            __Pyx_SET_SIZE(copy, -Py_SIZE(copy));
```